onnxruntime/core/framework/session_state.cc

```diff
@@ -242,7 +239,25 @@ const SessionState::NameNodeInfoMapType& SessionState::GetInputNodeInfoMap() con
 }
 
 void SessionState::AddOutputNameToNodeInfoMapping(const std::string& output_name, const NodeInfo& node_info) {
-  output_names_to_nodeinfo_mapping_[output_name].push_back(node_info);
+  auto& output_names_to_nodeinfo = output_names_to_nodeinfo_mapping_[output_name];
+  if (!output_names_to_nodeinfo.empty())
+    std::cout << output_name;
```

Comment:
cout?

Reply:
Debug code that shouldn't be there. Thanks for catching it. Removed.